### DISCUSSION TITLE
使用Trojan协议访问Cloudflare反向代理的网站被禁止访问

### DISCUSSION THREAD

**Discussion by universonic** [v2fly/v2ray-core#2293] [General]
我在尝试访问[ChatGPT](https://chat.openai.com/chat)时发现了这个问题。我试了美国和日本的AWS的多个节点，也尝试过换IP和端口，担心是Cloudflare侧有侦测，但是还是每次只能用一小会儿，很快就会被Cloudflare的反向代理阻断 `Access denied
Error code 1020`。并且我还观察到有其他几个使用了Cloudflare服务的网站也出现了这个问题。大家有什么应对办法吗？

**Comment by MakeFortuneQuietly** [comment]
套上CF后的trojan走的是什么传输协议？ 
tcp不行把，ws和grpc应该可以

**Reply by universonic** [reply]
不是v2ray用的Cloudflare, 而是通过v2ray访问的目标网站使用了Cloudflare

**Comment by weiidii** [comment]
遇到同样问题，同问～

**Comment by FWiner** [comment]
我也有这个怀疑

**Comment by FWiner** [comment]
> 我在尝试访问[ChatGPT](https://chat.openai.com/chat)时发现了这个问题。我试了美国和日本的AWS的多个节点，也尝试过换IP和端口，担心是Cloudflare侧有侦测，但是还是每次只能用一小会儿，很快就会被Cloudflare的反向代理阻断 `Access denied Error code 1020`。并且我还观察到有其他几个使用了Cloudflare服务的网站也出现了这个问题。大家有什么应对办法吗？

@universonic 怎么样解决了么，我最近换节点，换ip还是不行

**Comment by AkinoKaede** [comment]
This is because your proxy IP is blocked from accessing these sites, and it has nothing to do with Trojan.
